lib/assessment.rb

```diff
@@ -13,36 +13,47 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(&:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any? { |x| x.odd? }
```

Comment:
Correct. Alternatively,
`enum.any?(&:odd?)` 
(or, for Ruby 2.0 and above, `enum.any?(:odd?)`)